dpe/src/response.rs

```diff
@@ -105,6 +114,7 @@ pub struct NewHandleResp {
 pub struct DeriveChildResp {
     pub handle: ContextHandle,
     pub parent_handle: ContextHandle,
+    pub resp_hdr: ResponseHdr,
```

Comment:
To make the serialization correct I think resp_hdr needs to be the first field in each of these structs